libs/connector/src/secretarium.connector.ts

```diff
@@ -305,6 +305,8 @@ export class SCP {
                     const cryptoKey = await crypto.subtle!.importKey('raw', key, { name: 'AES-GCM' }, false, ['encrypt', 'decrypt']);
                     this._session = new SCPSession(iv, cryptoKey);
 
+                    if (userKey === undefined) userKey = await Key.createKey();
+
```

Comment:
We can't exactly to this with this package because SCP v1 requires a user defined Key as per its specs (and because @bfoing would pop someone's head off 😜)
Now this is basically what we would end-up doing once SCP v2 is implemented.